client/src/consts/theme.tsx

```diff
@@ -1,9 +1,31 @@
 const colors = {
   gray100: "#393838",
+  gray200: "#525252",
+  gray300: "#6e6e6e",
+  black100: "#343434",
+  black200: "#34343",
+  black300: "#03030363",
+  black400: "#131313",
 };
 
 const theme = {
   button: {
     backgroundColor: colors.gray100,
   },
+  NotificationTransform: {
+    backgroundColor: colors.black100,
+  },
+  DropDownContainer: {
+    backgroundColor: colors.black200,
+  },
+  DropDownShelf: {
+    boxShadow: colors.black300,
+    border: colors.gray200,
+    backgroundColor: colors.black100,
+  },
+  DropDownIconActiveStyle: {
+    border: colors.gray200,
+    filter: colors.black400,
+    backgroundColor: colors.gray300,
+  },
```

Comment:
eerrm, feel free to nest stuff. Much more readable and scales muuuch better.

Let's do the same with notifications colors.

Also, let's use lowerCamelCase instead of UpperCamelCase.

UpperCamelCase is reserved for components names.

This is implicit convention between FE devs and helps readability. No great reason behind it. Just a convention to help readability.

```suggestion
  dropdown: {
    container: {
      backgroundColor: colors.black200,
    },
    shelf: {
      boxShadow: colors.black300,
      border: colors.gray200,
      backgroundColor: colors.black100,
    },
    iconActiveStyle: {
      border: colors.gray200,
      filter: colors.black400,
      backgroundColor: colors.gray300,
    },
  },
```